.travis.yml

```diff
@@ -65,10 +66,6 @@ jobs:
           name: "Fedora 31"
           env:
             - FROM='fedora:31'
-        - os: linux
-          name: "Fedora 30"
-          env:
-            - FROM='fedora:30'
```

Comment:
We should add newer versions when removing older versions.

Reply:
Is Fedora 33 out yet? It doesn't look like it: https://fedorapeople.org/groups/schedule/f-33/f-33-key-tasks.html

Reply:
Fedora likes to only keep 2 active versions the with the upstream repo always called Rawhide

Reply:
okay - so we should only have 2 active versions in support at any given time. I don't use Fedora myself so just trying to make sure we're keeping things in the supported versions appropriately. Going forward, I'd expect we drop one and add one at roughly the same time.

Reply:
Fedora is Red Hat's experimental test bed what works gets added to official RH releases and then the Centos project re-brands that